edenai_apis/apis/klippa/klippa_api.py

```
@@ -2,10 +2,12 @@
 from io import BufferedReader, BytesIO
 from json import JSONDecodeError
 from typing import Dict
+import aiofiles
 
 import aiofiles
```

Ruff (0.14.5)

Redefinition of unused `aiofiles` from line 5

Remove definition: `aiofiles`

(F811)